src/connect4/core.rs

```diff
@@ -525,16 +494,25 @@ pub struct GameState {
     mouse_disabled: bool,
     gameover: bool,
     pub reset_button: Button,
-
+    pub main_menu_button: Button,
 }
 
 //Implementation based on structure in example from GGEZ repo (see https://github.com/ggez/ggez/blob/master/examples/02_hello_world.rs)
 impl GameState {
     pub fn new(ctx: &mut Context, players: i32) -> GameState {
         let board_pos = BOARD_POS_OFFSET;
-        let text = graphics::Text::new("Reset");
-        let text_width = text.width(ctx) as f32;
-        let text_height = text.height(ctx) as f32;
+        let main_menu_btn_text = graphics::Text::new(("Main Menu", graphics::Font::default(), 16f32,));
+        let main_menu_text_width = main_menu_btn_text.width(ctx) as f32;
+        let main_menu_text_height = main_menu_btn_text.height(ctx) as f32;
+        let main_menu_btn_outline = graphics::Rect::new(RESET_BUTTON_OFFSET.0 as f32, RESET_BUTTON_OFFSET.1 as f32 + main_menu_text_height, main_menu_text_width, main_menu_text_height);
+        let mut main_menu_btn = Button::new(main_menu_btn_text, main_menu_btn_outline);
+
```

Comment:
For what it's worth I would have considered moving this code plus the reset button stuff into a new method like `create_buttons` if we were working longer, but this is completely fine